CONTRIBUTING.md

```diff
@@ -58,13 +58,14 @@ On Windows you should be able to build the plugins inside IntelliJ as described
 
 To work in IDEA, just "Open" the cloud-tools-for-intellij directory 
 (the root directory cloned from Github) from the IDEA opening screen.
+To build the plugins, select "Prepare All Plugin Modules For Deployment"
```

Comment:
How about we confine this instruction to the build section where the Windows specific mention is? It's not really specific to project importing anyway.